airflow/providers/amazon/aws/secrets/secrets_manager.py

```diff
@@ -0,0 +1,125 @@
+#
```

Comment:
I think it needs rebase - those are unrelated changes.

Reply:
Done :)